src/main/java/io/r2dbc/postgresql/client/BalancedResolverGroup.java

```diff
@@ -0,0 +1,21 @@
+package io.r2dbc.postgresql.client;
```

Comment:
Nit: Please add a license header (see other files) with 2020 as the inception year.